staking/types/validator.go

```diff
@@ -183,8 +185,10 @@ type VoteWithCurrentEpochEarning struct {
 
 // ValidatorStats to record validator's performance and history records
 type ValidatorStats struct {
-	// APR ..
-	APR numeric.Dec `json:"-"`
+	// APRs is the APR history
+	APRs []numeric.Dec `json:"-"`
```

Comment:
I think it's better to use a map from epoch to apr. This current way is not flexible in terms of usage.

Reply:
done.